django-backend/fecfiler/reports/tests/test_models.py

```diff
@@ -30,3 +30,18 @@ def test_amending_f24(self):
             f24_report.form_24.original_amendment_date, new_upload_submission.created
         )
         self.assertEquals(f24_report.form_type, "F24A")
+
+    def test_delete(self):
+        f24_report = Report.objects.get(id="10000f24-d926-4e89-ad4b-000000000001")
+        f24_report.delete()
+
```

Comment:
Can we add assertions to test that transactions are not soft deleted for an F24 and are soft deleted otherwise?